script.js

```diff
@@ -0,0 +1,27 @@
+async function fetchTodo() {
+    try {
+        const response = await fetch('https://jsonplaceholder.typicode.com/todos');
+        if (!response.ok) {
+            throw new Error('Network response was not ok');
+        }
+        const json = await response.json();
+        return json
+    } catch (error) {
+        console.error('There was a problem with the fetch operation:', error);
+    }
+}
+
+ async function appendData() {
+    const data =  await fetchTodo();
```

Comment:
`appendData` will take data as a parameter and perform its task; it is not responsible for fetching the data.